### PR TITLE
[SDK] Consolidate more tests - reduce contract deployment

### PR DESCRIPTION
**Pull request by kien-ngo**
## Problem solved

Short description of the bug fixed or feature added

---

## PR-Codex overview
This PR primarily focuses on updating tests related to the `ERC721` and `ERC1155` contracts, including renaming and restructuring tests, adding new assertions, and removing obsolete tests.

### Detailed summary
- Deleted tests for `mintTo` and `deployERC721` functionality.
- Updated contract names in tests from "Test DropERC1155" to "EditionDrop" and "Test TokenERC1155" to "Edition".
- Added assertions to check deployed contract names.
- Introduced new tests for minting NFTs with different input types.
- Removed reliance on secret keys in certain tests.

> ✨ Ask PR-Codex anything about this PR by commenting with `/codex {your question}`

### CODE REVIEW

**Comment by kien-ngo** [comment]
* **#5258** <a href="https://app.graphite.dev/github/pr/thirdweb-dev/js/5258?utm_source=stack-comment-icon" target="_blank">Graphite</a> 👈
* `main`

This stack of pull requests is managed by Graphite. <a href="https://stacking.dev/?utm_source=stack-comment">Learn more about stacking.</a>
<h2></h2>

 Join @kien-ngo and the rest of your teammates on <a href="https://graphite.dev?utm-source=stack-comment">Graphite <b>Graphite</b></a>

File: packages/thirdweb/src/extensions/erc1155/drop1155.test.ts [hunk 1]
```diff
@@ -45,12 +46,21 @@ describe.runIf(process.env.TW_SECRET_KEY)(
         chain: ANVIL_CHAIN,
         client: TEST_CLIENT,
         params: {
-          name: "Test DropERC1155",
-          contractURI: TEST_CONTRACT_URI,
+          name: "EditionDrop",
```
Comment (joaquim-verges) [inline packages/thirdweb/src/extensions/erc1155/drop1155.test.ts:49]: Can You still pass a contract uri here? Was useful to skip the ipsy upload

File: packages/thirdweb/src/extensions/erc1155/token1155.test.ts [hunk 2]
```diff
@@ -37,12 +38,21 @@ describe.runIf(process.env.TW_SECRET_KEY)("TokenERC1155", () => {
       chain: ANVIL_CHAIN,
       client: TEST_CLIENT,
       params: {
-        name: "Test TokenERC1155",
-        contractURI: TEST_CONTRACT_URI,
+        name: "Edition",
```
Comment (joaquim-verges) [inline packages/thirdweb/src/extensions/erc1155/token1155.test.ts:41]: Same here. Also this tests the exact same thing no? Doesn't actually help coverage

File: packages/thirdweb/src/extensions/erc721/token721.test.ts [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+
+  it("isGetNFTsSupported should work", async () => {
+    const abi = await resolveContractAbi<Abi>(token721Contract);
+    const selectors = abi
+      .filter((f) => f.type === "function")
+      .map((f) => toFunctionSelector(f));
+    expect(isGetNFTSupported(selectors)).toBe(true);
+  });
+
+  it("should return a default value if the URI of the token doesn't exist", async () => {
```
Comment (joaquim-verges) [inline packages/thirdweb/src/extensions/erc721/token721.test.ts:81]: What's the value of moving these tests around? Feels like unnecessary work. Would you rather focus on actually writing new tests on areas that aren't covered.

You can check the uncovered areas on codecov, any file/folder with low coverage is what you should focus on. 

All of react for example is particularly low

File: packages/thirdweb/src/extensions/erc1155/drop1155.test.ts (hunk 1 above)
Comment (kien-ngo) [reply packages/thirdweb/src/extensions/erc1155/drop1155.test.ts:49]: oops

Comment (kien-ngo) [reply packages/thirdweb/src/extensions/erc1155/drop1155.test.ts:49]: added back

File: packages/thirdweb/src/extensions/erc1155/token1155.test.ts (hunk 2 above)
Comment (kien-ngo) [reply packages/thirdweb/src/extensions/erc1155/token1155.test.ts:41]: added back

**Review by gregfromstl — APPROVED** [review]

**Comment by gregfromstl** [comment]
### Merge activity

* **Nov 1, 2:08 PM EDT**: The merge label 'merge-queue' was detected. This PR will be added to the [Graphite merge queue](https://app.graphite.dev/merges?org=thirdweb-dev&repo=js) once it meets the requirements.

**Pull request merged**